### ISSUE TITLE
Forum does not support TAB or ESC functionality

### ISSUE THREAD

**Issue by JManion32** [Submitty/Submitty#11801]
## Describe the bug

Above text areas in the forum, there is an accessibility message that says _"Press TAB to indent. Press ESC to advance from answer area."_  However, when I press `TAB`, it uses its default behavior and switches focus to another element lower on the page. When I press `ESC`, nothing happens.

## Expected behavior

**There are 2 Solutions:**
1. Pressing `TAB` should not use default behavior, and instead insert a tab character into the text area. `ESC` should be used to switch focus instead. A potential issue with here is that the DOM's undo/redo stack would likely be disrupted.
2. Alter accessibility message to state actual functionality.

## To Reproduce
1. Create or Edit a thread in the discussion forum
2. See _"Press TAB to indent. Press ESC to advance from answer area."_  above the text area.
3. Attempt to use `TAB` or `ESC`
4. Observe that result does not match what is intended

**Configuration**
 - OS: Windows 11
 - Browser Chrome and Firefox

## Video
In this video, I press `TAB` several times, then navigate back to the text area and press `ESC` several times.

https://github.com/user-attachments/assets/ba481a63-9ddc-4655-a2df-2dd726aebb40

**Comment by SahilBijlani** [comment]
I would love to work on this issue! can u assign it to me?
